packages/world/ts/config/resolveWorldConfig.ts

```diff
@@ -57,8 +59,18 @@ export function resolveWorldConfig(
  * Default value for accessListAddresses is []
  * Default value for accessListSystems is []
  */
-export function resolveSystemConfig(systemName: string, config?: SystemConfig, existingContracts?: string[]) {
-  const name = config?.name ?? systemName;
+export function resolveSystemConfig(
+  systemName: string,
+  configNamespace: string,
+  config?: SystemConfig,
+  existingContracts?: string[],
```

Comment:
can we convert this to an object? it gets really hard to reason about past 2 args

Reply:
done!